dev/archetypes/maven-java/src/main/resources/archetype-resources/__service1Name__-api/src/main/java/__packageInPathFormat__/__service1Name__/api/GreetingMessage.java

```diff
@@ -1,3 +1,6 @@
+/*
+ * Copyright (C) 2017 Lightbend Inc. <https://www.lightbend.com>
+ */
```

Comment:
I ran `; createHeaders ; test:createHeaders` in sbt, and it added headers to a bunch of source files that didn't have them previously (I guess because, by default, it only adds headers when compiling a file).

Do we want them there? Maybe it doesn't make sense to put our copyright into archetype files, though previously some did, others didn't.

If we don't want them, we can configure the project to exclude them strip them out.